ui/src/components/list-view/QasListView.stories.js

```diff
@@ -67,6 +67,14 @@ export default {
       description: 'Ignore entity and specify another endpoint.'
     },
 
+    useInfiniteScroll: {
+      description: 'Use pagination in infinite scroll mode.'
```

Comment:
`Disable pagination and use infinite scroll mode.`

Reply:
mas ele não desabilita, ele habilita

Reply:
Não entendi